IDE Cell run documentation (#1858)
astronomer/docs: astro/cloud-ide/quickstart.md

The **Logs** tab contains all logs generated by the cell run, including Airflow logs and Python errors. The **Results** tab contains the contents of your Python console. Click **Results** to view the result of your successful cell run.

:::info 

> If you receive the error `Could not connect to cell execution environment` after running a cell, check the [Astro status page](https://status.astronomer.io/) to see whether the Astro control plane is currently experiencing issues. If the control plane is fully operational, contact [Astronomer support](https://cloud.astronomer.io/support) and share the error. Astronomer support might need to set up additional cloud infrastructure for the IDE to enable cell runs. 

Reviewer: ```suggestion
If the error message `Could not connect to cell execution environment` appears after running a cell, check the [Astro status page](https://status.astronomer.io/) to determine the operational status of the Astro control plane. If the control plane is operational, contact [Astronomer support](https://cloud.astronomer.io/support) and share the error. To enable cell runs, Astronomer support might need to set up additional cloud infrastructure for the IDE. 
```
